sale_order_customer_free_ref/tests/test_sale_order_customer_free_ref.py

```diff
@@ -1,10 +1,10 @@
 # Copyright 2021 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
-from odoo.tests.common import SingleTransactionCase
+from odoo.addons.base.tests.common import BaseCommon
 
 
-class TestSaleOrderCustomerFreeRef(SingleTransactionCase):
+class TestSaleOrderCustomerFreeRef(BaseCommon):
```

Comment:
FTR this is not the same. `SingleTransactionCase` runs all tests in one transaction. Now, I don't see why we should use a single trans here.... so that's ok.
However, by using `BaseCommon` you can remote the line 11.